jaide/cli.py

```diff
@@ -298,8 +298,10 @@ def main(ctx, host, password, port, quiet, session_timeout, connect_timeout,
               "--at and --confirm are mutually exclusive, and confirm will"
               " override. Can be in one of two formats: hh:mm[:ss]  or  "
               "yyyy-mm-dd hh:mm[:ss]")
+@click.option('-f', '--format', default='set', help="Specify set or stanza mode for the configuration")
+@click.option('-a', '--action', default='merge', help='If using stanza config, action can be merge or override')
```

Comment:
on these help lines, maybe we should wrap the options in quotes to make it more obvious. Also, a callback for validating the option they give?